packages/venia-ui/lib/components/AccountChip/accountChip.css

```diff
@@ -0,0 +1,11 @@
+.root {
+    align-items: center;
+    column-gap: 0.5rem;
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-columns: auto 1fr;
+}
+
+.loader {
+    composes: indicator from '../LoadingIndicator/indicator.css';
```

Comment:
At first, I was thinking why didn't Andy use `../LoadingIndicator/indicator.css` directly in the component. Then I realized this is a better way because we are providing a default and also letting others extend the `loader` class if needed using the prop classes. Good one 👍 .

Reply:
Thanks! And yeah, I originally did have the `LoadingIndicator` in there but in addition to the benefits you mentioned I didn't want all the extra spacing (margins and padding) baggage that came with it.